### DISCUSSION TITLE
Web UI/Theme Modifications

### DISCUSSION THREAD

**Discussion by sumitk21** [wiz64/saavn-web-ui#1] [Ideas]
Hello,
The webapp is undoubtly fast and quick.
But, The UI doesn't look that good as a dark background and dark gradient background doesn't fit.
Make the UI a bit brighter with clean light background, or just simple dark with some green.

**Comment by wiz64** [comment]
going to have a spotify-like dark theme and a light focused theme, with a toggle button.
Will include in v2.
Thanks for your valuable feedback !

**Reply by sumitk21** [reply]
> going to have a spotify-like dark theme and a light focused theme, with a toggle button. Will include in v2. Thanks for your valuable feedback !

Okay, Thanks
